2_observations.yml

```diff
@@ -74,8 +84,18 @@ targets:
   2_observations/out/obs_flow_drb.rds.ind:
     command: munge_flow(
       out_ind = target_name,
-      dat_ind = '2_observations/out/drb_discharge_daily_dv.csv.ind',
+      dat_ind = '2_observations/in/daily_flow.rds.ind',
```

Comment:
Should we delete the `2_observations/out/drb_discharge_daily_dv.csv` target in `getters.yml` too?